crmsh/service_manager.py

```diff
@@ -0,0 +1,100 @@
+import typing
+
+import crmsh.parallax
+import crmsh.sh
+
+
+class ServiceManager(object):
+    """
+    Class to manage systemctl services
+    """
+
+    def __init__(self, shell: crmsh.sh.AutoShell = None):
+        if shell is None:
+            self._shell = crmsh.sh.auto_shell()
+        else:
+            self._shell = shell
```

Comment:
```Python
self._shell = shell or crmsh.sh.auto_shell()
```

Reply:
I think `crmsh.sh.auto_shell() if shell is None or shell` is better, as it is more explicit.